WalletWasabi.Fluent/Controls/DialogContentHost.cs

```diff
@@ -0,0 +1,46 @@
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Controls.Primitives;
+
+namespace WalletWasabi.Fluent.Controls
+{
+	/// <summary>
+	/// Manages and hosts dialogs when it's bound to <see cref="IDialogHost"/> objects.
+	/// </summary>
+	public class DialogContentHost : TemplatedControl
+	{
+		public static readonly StyledProperty<object> ContentProperty =
+			AvaloniaProperty.Register<DialogContentHost, object>(nameof(Content));
+
+		public static readonly StyledProperty<bool> IsDialogOpenProperty =
+			AvaloniaProperty.Register<DialogContentHost, bool>(nameof(IsDialogOpen));
+
+		static DialogContentHost()
+		{
+			IsDialogOpenProperty.Changed.AddClassHandler<DialogContentHost>(UpdatePseudoClasses);
+		}
+
+		private static void UpdatePseudoClasses(DialogContentHost arg1, AvaloniaPropertyChangedEventArgs arg2)
+		{
+			arg1.PseudoClasses.Set(":open", (bool)arg2.NewValue);
+		}
```

Comment:
Move it after properties to fix CodeFactor.

Reply:
Done.